openvpn_api/vpn.py

```diff
@@ -67,6 +93,8 @@ def disconnect(self, _quit=True) -> None:
         if self._socket is not None:
             if _quit:
                 self._socket_send("quit\n")
+
+            self._socket_file.close()
             self._socket.close()
             self._socket = None
```

Comment:
You're spawning threads up but they never get torn down. If we call disconnect, we should also stop the threads

Reply:
I partially take that back, they do break when `self.is_connected` is False but the're no waiting for shutdown in the disconnect method